tests/integration/cmd_storage_test.go

```diff
@@ -0,0 +1,139 @@
+package integration
+
+import (
+	"os"
+	"path/filepath"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+)
+
+var _ = Describe("odo storage command", func() {
+	var project string
+	var context string
+
+	// This is run after every Spec (It)
+	var _ = BeforeEach(func() {
+		SetDefaultEventuallyTimeout(10 * time.Minute)
+		os.Setenv("GLOBALODOCONFIG", filepath.Join(context, "config.yaml"))
+		project = helper.CreateRandProject()
+		context = helper.CreateNewContext()
+		oc = helper.NewOcRunner("oc")
+	})
+
+	// Clean up after the test
+	// This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.DeleteProject(project)
+		helper.DeleteDir(context)
+		os.Unsetenv("GLOBALODOCONFIG")
+	})
+
+	Context("when running help for storage command", func() {
+		It("should display the help", func() {
+			appHelp := helper.CmdShouldPass("odo", "storage", "-h")
+			Expect(appHelp).To(ContainSubstring("Perform storage operations"))
+		})
+	})
+
+	Context("when running storage command without required flag(s)", func() {
+		It("should fail", func() {
+			helper.CopyExample(filepath.Join("source", "dotnet"), context)
+			helper.CmdShouldPass("odo", "component", "create", "dotnet", "dotnet", "--app", "dotnetapp", "--project", project, "--context", context)
+			stdErr := helper.CmdShouldFail("odo", "storage", "create", "pv1")
+			Expect(stdErr).To(ContainSubstring("Required flag"))
+			//helper.CmdShouldFail("odo", "storage", "create", "pv1", "-o", "json")
```

Comment:
@surajnarwade @mik-dass Can you please confirm if command fails then does it capture the output in json format ?

Reply:
we dont have control over this as of now :|